LION/models/LPD.py

```diff
@@ -176,8 +194,8 @@ def __init__(
     def default_parameters():
         LPD_params = Parameter()
         LPD_params.n_iters = 10
-        LPD_params.data_channels = [7, 32, 32, 5]
-        LPD_params.reg_channels = [6, 32, 32, 5]
+        LPD_params.data_channels = [7, 32, 32, 32, 5]
+        LPD_params.reg_channels = [6, 32, 32, 32, 5]
```

Comment:
You added more channels to LPD. 

This is fine for the paper ofc, but any reason why?
Should we have a `cLPD_paper_parameters` maybe? The original LPD I think has the others.

Reply:
It's just to match the cLPD parameters for the paper. Adding cLPD_paper_parameters sounds good!